Sharprompt/ConsoleScope.cs

```diff
@@ -95,5 +104,81 @@ public void Render<TModel>(Action<ConsoleRenderer, TModel> template, TModel mode
 
             Console.CursorVisible = _cursorVisible;
         }
+
+        private static void CancelStandardInput()
+        {
+            const int STD_INPUT_HANDLE = -10;
+            CancelIoEx(GetStdHandle(STD_INPUT_HANDLE), IntPtr.Zero);
```

Comment:
I think it works on Windows, but it probably doesn't work on Linux / macOS.

Reply:
I'll do some research and tests how this can be achieved on linux. Seems to be not as straight forwards as on windows.